docs/usage/guides/bundler/anti-bloat.md

```diff
@@ -65,7 +65,7 @@ UPX, **Ultimate Packer for eXecutables**, is a dinosaur amongst the binary packe
 Basically it compresses the binary and decompresses it at runtime. It should work for pretty much any binary type out there. Read more: https://github.com/upx/upx
 
 <Alert type="warning" title="Warning" icon="alert">
-You should know that this technique might flag your binary as a virus on Windows and macOS - so use at your own discretion, and as always validate with Frida and do real distribution testing!
+You should know that this technique might flag your binary as a virus on Windows and macOS - so use at your own discretion, and as always validate with [Frida](https://github.com/frida/frida-rust) and do real distribution testing!
```

Comment:
Probably best to link to the official Rust binding instead of it's docs?
Or do we instead link right to the good stuff that explains Frida very quickly and the why at https://frida.re/docs/home/ ?

Reply:
Why not just the homepage?

Reply:
updated my comment above as to why potentially.

Reply:
In this case, I guess consumers are free to use whatever they want, not necessarily Frida crate. I've never used it myself but I bet they have a standalone CLI, too.

Reply:
Agreed, then I think the first page of Frida docs is perfect here, let me change that.  Besides, folks clicking on a link are typically clicking on it in the first place because of "Hmm, I never heard of this name, let me click to find out and learn more".  And crates are opinionated as you say and multiple.